### ISSUE TITLE
Loading bar stuck at around 60% after starting pathfinder wotr with mods enabled.

### ISSUE THREAD

**Issue by lorddiablo7** [Vek17/TabletopTweaks-Core#29]
**Describe the bug**
Loading bar stuck at around 60% after starting pathfinder wotr with mods enabled. Using TabletopTweaks-Core, TabletopTweaks-Base and TabletopTweaks-MythisReworks with UnityModManager. Latest versions of all. When not using Core, the game runs properly

**To Reproduce**
Just start the game with mods enabled in UMM

**Expected behavior**

**Screenshots**

**Additional context**

**Comment by Vek17** [comment]
On current version of the game with current version of everything, it all should load fine and does in my test environment. 

Thing to ensure:
Game version 2.1.4w
TTT-Core Version: 0.6.7
TTT-Base Version: 2.5.13
TTT-Reworks Version: 1.2.7

To debug further I would need to see your logfiles, specifically player.log at %appdata%\..\LocalLow\Owlcat Games\Pathfinder Wrath Of The Righteous

**Comment by lorddiablo7** [comment]
[GameLog.txt](https://github.com/Vek17/TabletopTweaks-Core/files/11924992/GameLog.txt)
[GameLogFull.txt](https://github.com/Vek17/TabletopTweaks-Core/files/11924993/GameLogFull.txt)
[GameLogFullPrev.txt](https://github.com/Vek17/TabletopTweaks-Core/files/11924994/GameLogFullPrev.txt)
[GameLogPrev.txt](https://github.com/Vek17/TabletopTweaks-Core/files/11924995/GameLogPrev.txt)
[Player.log](https://github.com/Vek17/TabletopTweaks-Core/files/11924997/Player.log)
[Player-prev.log](https://github.com/Vek17/TabletopTweaks-Core/files/11924998/Player-prev.log)

These are the log files I found. 
The game as well as the TTT mods were all updated to the latest versions.
I also checked using previous versions but it always gets stuck at 60% on loading screen.

**Comment by lorddiablo7** [comment]
[GameLog.txt](https://github.com/Vek17/TabletopTweaks-Core/files/11925036/GameLog.txt)
[GameLogFull.txt](https://github.com/Vek17/TabletopTweaks-Core/files/11925037/GameLogFull.txt)
[GameLogFullPrev.txt](https://github.com/Vek17/TabletopTweaks-Core/files/11925038/GameLogFullPrev.txt)
[GameLogPrev.txt](https://github.com/Vek17/TabletopTweaks-Core/files/11925039/GameLogPrev.txt)
[Player.log](https://github.com/Vek17/TabletopTweaks-Core/files/11925040/Player.log)
[Player-prev.log](https://github.com/Vek17/TabletopTweaks-Core/files/11925041/Player-prev.log)

these are for TTT-Core 0.6.7
all latest softwares

**Comment by Vek17** [comment]
Your failure is not in Tabletop tweaks it is in SparsSorcerousSundries
```
MissingMethodException: void Kingmaker.Blueprints.JsonSystem.BlueprintsCache.AddCachedBlueprint(Kingmaker.Blueprints.BlueprintGuid,Kingmaker.Blueprints.SimpleBlueprint)
  at BlueprintCore.Utils.BlueprintTool.Create[T] (System.String name, System.String guid) [0x0002a] in <b7969400d95148dfaf274bbc59666594>:0 
  at BlueprintCore.Blueprints.Configurators.Items.SharedVendorTableConfigurator.New (System.String name, System.String guid) [0x00000] in <b7969400d95148dfaf274bbc59666594>:0 
  at SparsSorcerousSundries.Items.Gilmore.CreateGilmore () [0x00001] in <b7969400d95148dfaf274bbc59666594>:0 
  at SparsSorcerousSundries.Items.CustomItemInjector.CustomItemInjection () [0x00001] in <b7969400d95148dfaf274bbc59666594>:0 
  at (wrapper dynamic-method) Kingmaker.Blueprints.JsonSystem.BlueprintsCache.Kingmaker.Blueprints.JsonSystem.BlueprintsCache.Init_Patch106(Kingmaker.Blueprints.JsonSystem.BlueprintsCache)
  at (wrapper dynamic-method) Kingmaker.Blueprints.JsonSystem.StartGameLoader.Kingmaker.Blueprints.JsonSystem.StartGameLoader.LoadPackTOC_Patch2(Kingmaker.Blueprints.JsonSystem.StartGameLoader)
  at Kingmaker.GameStarter+<StartGameCoroutine>d__25.MoveNext () [0x00375] in <cac0edc92d5645deb2aa8e59e70a8f59>:0 
  at UnityEngine.SetupCoroutine.InvokeMoveNext (System.Collections.IEnumerator enumerator, System.IntPtr returnValueAddress) [0x00026] in <25a85da7c6f04932b86e339dfd12957d>:0 
```

**Comment by lorddiablo7** [comment]
Could you please tell me how do I fix this problem?
I am not very well versed in this matter !
I'm sorry if I'm taking a lot of your time .

**Comment by Vek17** [comment]
That mod needs to be recompiled againt an up to date blueprint core. Not
sure if they or anyone else has already done so.

**Comment by lorddiablo7** [comment]
Okay .
Thank you soo much.
The game works now .

**Comment by DimazikSkyter** [comment]
Sry, anyone fix and recompile? Get this bug at latest version

**Comment by haoknowah** [comment]
Same. 
Now it is not installing on Unity Mod Manager. I tried updating it and I uninstalled it in the process now the status won't change on the mod manager. I can right click and install or click install and select the zip but it will say that it finished unpacking and not do anything.

**Comment by Vek17** [comment]
Update to the latest ttt-base

**Comment by haoknowah** [comment]
I figured it out. It wasn't liking the fact that I chose the second zip download option on the release page instead of the first that has the image of the box next to it.

**Comment by whl970831** [comment]
Has anyone ever encountered this type of problem?

My game testing environment:
Game version 2.1.5r
TTT-Core Version: 0.6.7
TTT-Base Version: 2.5.14
It always gets stuck in 60% of the loading screen

Here are the log files

[GameLog.txt](https://github.com/Vek17/TabletopTweaks-Core/files/12367779/GameLog.txt)
[GameLogFull.txt](https://github.com/Vek17/TabletopTweaks-Core/files/12367783/GameLogFull.txt)
[GameLogFullPrev.txt](https://github.com/Vek17/TabletopTweaks-Core/files/12367787/GameLogFullPrev.txt)
[GameLogPrev.txt](https://github.com/Vek17/TabletopTweaks-Core/files/12367794/GameLogPrev.txt)
[Player.log](https://github.com/Vek17/TabletopTweaks-Core/files/12367797/Player.log)
[Player-prev.log](https://github.com/Vek17/TabletopTweaks-Core/files/12367799/Player-prev.log)


It is worth mentioning that I found that downloading all files through the code port can enter the game interface, but downloading through the Releases port will cause a lag of 60%
（I'm wondering if the blueprint needs to be recompiled due to the new version of the game）

**Comment by whl970831** [comment]
> I figured it out. It wasn't liking the fact that I chose the second zip download option on the release page instead of the first that has the image of the box next to it.

Yes, I tried using the 0.6.7 version of the Source code (zip) in Releases to solve this problem. The good news is that it solved the stuttering in the 60% progress bar, while the bad news is that it still doesn't work, and the display is still

this figure：
![image](https://github.com/Vek17/TabletopTweaks-Core/assets/38749628/55b3bd8d-9893-4c98-b7cb-9b3890e63d27)

**Comment by whl970831** [comment]
> I figured it out. It wasn't liking the fact that I chose the second zip download option on the release page instead of the first that has the image of the box next to it.

es, bor. I tried downloading and using version 0.6.7 of the source code (zip) in Release to solve this problem. The good news is that it solves the lag in 60% of the progress bar, while the bad news is that it still doesn't work and the display is still

As shown in the figure:
![image](https://github.com/Vek17/TabletopTweaks-Core/assets/38749628/dec66bff-918a-44ac-8f6f-dae0ca58f1f7)

**Comment by whl970831** [comment]
> I figured it out. It wasn't liking the fact that I chose the second zip download option on the release page instead of the first that has the image of the box next to it.

yes, bor. I tried downloading and using version 0.6.7 of the source code (zip) in Release to solve this problem. The good news is that it solves the lag in 60% of the progress bar, while the bad news is that it still doesn't work and the display is still

As shown in the figure:
![image](https://github.com/Vek17/TabletopTweaks-Core/assets/38749628/25c1a536-cf31-4e67-a6dd-cb18b33844b7)

My game testing environment:
Game version 2.1.5r
TTT-Core Version: 0.6.7
TTT-Base Version: 2.5.14

**Comment by Vek17** [comment]
Don't download the source code, download the release file 
https://github.com/Vek17/TabletopTweaks-Core/releases/download/v0.6.7/TabletopTweaks-Core.zip

The source code is not the mod in a way the game can understand and is not meant for users, that exists for other developers.

**Comment by Vek17** [comment]
Your failure is also happening in MicroscopicContentExpansion, so update that if there is an update

**Comment by whl970831** [comment]
> Your failure is also happening in MicroscopicContentExpansion, so update that if there is an update

Okay, thank you very much, but I still have a question. How did you identify the issue with MicroscopicContentExpansion? Is it written in which log file?

**Comment by Vek17** [comment]
Yes, the stack trace at the end of the file is thus and points to MicroscopticContentExpansion

```
MissingFieldException: Field 'Kingmaker.UnitLogic.Mechanics.Components.AddInitiatorAttackWithWeaponTrigger.WaitForAttackResolve' not found.
  at TabletopTweaks.Core.Utilities.Helpers.Create[T] (System.Action`1[T] init) [0x00009] in <a62c9c81f34b416ebe7105c0aa469e36>:0 
  at TabletopTweaks.Core.Utilities.BlueprintExtentions.AddComponent[T] (Kingmaker.Blueprints.BlueprintScriptableObject obj, System.Action`1[T] init) [0x00007] in <a62c9c81f34b416ebe7105c0aa469e36>:0 
  at MicroscopicContentExpansion.NewContent.Spells.DeadlyJuggernaut+<>c__DisplayClass0_0.<AddDeadlyJuggernaut>b__1 (Kingmaker.UnitLogic.Buffs.Blueprints.BlueprintBuff bp) [0x00058] in <dd0222920b3546e39e8c717a51d15c28>:0 
  at TabletopTweaks.Core.Utilities.Helpers.CreateBlueprint[T] (TabletopTweaks.Core.ModLogic.ModContextBase modContext, System.String name, System.Action`1[T] init) [0x00043] in <a62c9c81f34b416ebe7105c0aa469e36>:0 
  at MicroscopicContentExpansion.NewContent.Spells.DeadlyJuggernaut.AddDeadlyJuggernaut () [0x0003c] in <dd0222920b3546e39e8c717a51d15c28>:0 
  at MicroscopicContentExpansion.NewContent.AntipaladinFeatures.AntipaladinSpellbook.AddAntipaladinSpellbook () [0x0004c] in <dd0222920b3546e39e8c717a51d15c28>:0 
  at MicroscopicContentExpansion.NewContent.Antipaladin.AntipaladinAdder+BlueprintsCache_Init_Patch.Postfix () [0x0004a] in <dd0222920b3546e39e8c717a51d15c28>:0 
  at (wrapper dynamic-method) Kingmaker.Blueprints.JsonSystem.BlueprintsCache.Kingmaker.Blueprints.JsonSystem.BlueprintsCache.Init_Patch119(Kingmaker.Blueprints.JsonSystem.BlueprintsCache)
  at (wrapper dynamic-method) Kingmaker.Blueprints.JsonSystem.StartGameLoader.Kingmaker.Blueprints.JsonSystem.StartGameLoader.LoadPackTOC_Patch6(Kingmaker.Blueprints.JsonSystem.StartGameLoader)
  at Kingmaker.GameStarter+<StartGameCoroutine>d__25.MoveNext () [0x00375] in <1bdbbfc0e6fc4f2db83179edfe399546>:0 
  at UnityEngine.SetupCoroutine.InvokeMoveNext (System.Collections.IEnumerator enumerator, System.IntPtr returnValueAddress) [0x00026] in <4fae2288737c4dfda349689bbefabfcc>:0 
```
